packages/teleterm/src/services/config/providers/keyboardShortcutsConfigProvider.ts

```diff
@@ -0,0 +1,71 @@
+import { ConfigServiceProvider } from '../types';
+
+/**
+ * Modifier keys must be defined in the following order:
+ * Command-Control-Option-Shift for macOS
+ * Ctrl-Alt-Shift for other platforms
+ */
+
+export type KeyboardShortcutType =
+  | 'tab-1'
+  | 'tab-2'
+  | 'tab-3'
+  | 'tab-4'
+  | 'tab-5'
+  | 'tab-6'
+  | 'tab-7'
+  | 'tab-8'
+  | 'tab-9'
+  | 'tab-close'
+  | 'tab-new'
+  | 'tab-previous'
+  | 'tab-next'
+  | 'focus-global-search';
+
+export type KeyboardShortcutsConfig = Record<KeyboardShortcutType, string>;
+
+export const keyboardShortcutsConfigProvider: ConfigServiceProvider<KeyboardShortcutsConfig> =
+  {
+    provideDefaultsForPlatform(platform) {
```

Comment:
`getPlatformDefaults()` or event better `getDefaults(platform?: string)`

Reply:
Changed to `getDefaults(platform: string)` - we don't have to use `platform` parameter in the implementation, but it is always available.